packages/firebase_ui_auth/lib/src/views/email_link_sign_in_view.dart

```diff
@@ -82,6 +82,16 @@ class _EmailLinkSignInViewState extends State<EmailLinkSignInView> {
                 },
               ),
             ],
+            if (Navigator.canPop(context)) ...[
```

Comment:
I think a single check during state init is enough, no need to check on every rebuild. `late final canPop = Navigator.canPop(context);` defined on the state should do.